exporter/datadogexporter/utils/trace_helpers.go

```diff
@@ -132,3 +132,20 @@ func NormalizeServiceName(service string) string {
 
 	return s
 }
+
+// TruncateUTF8 truncates the given string to make sure it uses less than limit bytes.
+// If the last character is an utf8 character that would be splitten, it removes it
+// entirely to make sure the resulting string is not broken.
+func TruncateUTF8(s string, limit int) string {
```

Comment:
Can you add the URL for the source of this code on the Datadog Agent for reference?

Reply:
makes sense will do